Gemfile

```diff
@@ -90,6 +90,9 @@ end
 group :development, :test do
   gem 'awesome_print'
 
+  # Loads ENV variables from .env file in base folder
+  gem 'dotenv-rails'
```

Comment:
I personally like doing that but I wasn't sure if people in general like doing that.